app/views/layouts/govuk_publishing_components/application.html.erb

```diff
@@ -32,21 +32,22 @@
     </script>
     <% if @preview %>
       <main id="wrapper" class="govuk-width-container">
+        <%= yield %>
+      </main>
     <% else %>
       <%= render "govuk_publishing_components/components/layout_header", {
         environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
         product_name: GovukPublishingComponents::Config.component_guide_title,
         href: component_guide_path
       } %>
-
-      <main id="wrapper" class="govuk-width-container component-guide-wrapper">
-      <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: @guide_breadcrumbs  %>
-    <% end %>
-
-    <%= yield %>
-    </main>
-
-    <% unless @preview %>
+      <div class="govuk-width-container app-width-container--wide">
+        <% if @guide_breadcrumbs %>
+          <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: @guide_breadcrumbs  %>
+        <% end %>
+        <main id="wrapper" class="govuk-main-wrapper">
```

Comment:
Looks like the spacing above the component title has increased because of the `govuk-main-wrapper` class? It's a bit big.

<img width="585" alt="Screenshot 2020-02-14 at 15 21 40" src="https://user-images.githubusercontent.com/861310/74543718-ca120780-4f3d-11ea-81cb-66c6be93c5a8.png">

This seems to be tied to the replacement of the `component-guide-wrapper` class.

Reply:
Oh, this is because we're forcing a `margin-top` of 8 to the title component. Will have to update the component, as this would become a problem when updating frontend apps to the new layout too.

Reply:
Raised https://github.com/alphagov/govuk_publishing_components/pull/1302 to address this.